test/pom.xml

```diff
@@ -20,9 +20,8 @@
 	<dependencies>
 		<!-- This dependency is here because of the CustomHighDeliveryCountEsbJmsListener -->
 		<dependency>
-			<groupId>org.apache.geronimo.specs</groupId>
-			<artifactId>geronimo-jms_1.1_spec</artifactId>
-			<scope>provided</scope>
```

Comment:
Moeten we de scope `provided` niet handhaven?

Reply:
Doen we, staat in de managedDependencies van de parent pom